front/src/components/form/Checkbox.tsx

```diff
@@ -9,10 +9,17 @@ type OwnProps = {
   onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
 };
 
-const StyledContainer = styled.span`
+const StyledContainer = styled.div`
```

Comment:
I think we should create wrapper in another component. Otherwise all our checkboxes will have a 32 by 32 space around them.
Checkbox should remain a checkbox,
maybe we can create a TableCellCheckbox